src/pages/register-a-beacon/check-beacon-details.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { FunctionComponent } from "react";
+import { Grid } from "../../components/Grid";
+import { InsetText } from "../../components/InsetText";
+import { Layout } from "../../components/Layout";
+import { Button, BackButton } from "../../components/Button";
+import {
+  Form,
+  FormFieldset,
+  Input,
+  FormLegend,
+  FormLabel,
+  FormGroup,
+  Select,
+  SelectOption,
+  FormHint,
+} from "../../components/Form";
+import { Details } from "../../components/Details";
+import { A } from "../../components/Typography";
+
+const CheckBeaconDetails: FunctionComponent = () => (
+  <>
+    <Layout navigation={<BackButton href="/intent" />}>
+      <Grid
+        mainContent={
+          <>
+            <Form action="/register-a-beacon/check-beacon-summary">
+              <FormFieldset>
+                <FormLegend>Check beacon details</FormLegend>
+                <InsetText>
+                  The details of your beacon must be checked to ensure they have
+                  a UK encoding and if they are already registered with this
+                  service.
+                </InsetText>
+
+                <BeaconManufacturerSelect />
+
+                <BeaconModelSelect />
+
+                <BeaconHexIdInput />
+              </FormFieldset>
+              <Button buttonText="Submit" />
+            </Form>
+            <IfYouNeedHelp />
+          </>
+        }
+      />
+    </Layout>
+  </>
+);
+
+const BeaconManufacturerSelect: FunctionComponent = (): JSX.Element => (
+  <FormGroup>
+    <FormLabel htmlFor="beaconManufacturer">
+      Select your beacon manufacturer
+    </FormLabel>
+    <Select name="beaconManufacturer" defaultValue="default">
+      <option hidden disabled value="default">
+        Beacon manufacturer
+      </option>
```

Comment:
Sets initial value of dropdown to "Beacon manufacturer" but doesn't allow user to select it.  Means form does not choose a "real" beacon manufacturer as the default value; user must choose.